src/App.css

```
@@ -247,4 +405,44 @@ body {
+    width: 100%;
+  }
+
+  .blackout-stage {
+    height: 100vh;
+    min-height: 100vh;
+  }
+
+  .blackout-panel {
+    left: 12px;
+    right: 12px;
+    inset: 72px 12px 12px;
```

Stylelint (17.7.0)

[error] Overridden property "right" by shorthand "inset" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)